pkg/util/cloudproviders/cloudproviders.go

```diff
@@ -147,3 +147,23 @@ func GetPublicIPv4(ctx context.Context) (string, error) {
 	log.Infof("No public IPv4 address found")
 	return "", errors.New("No public IPv4 address found")
 }
+
+type cloudProviderAccountIDCollector struct {
+	name         string
+	metadataName inventories.AgentMetadataName
+	callback     func(context.Context) (string, error)
+}
+
+func CollectAccountID(ctx context.Context) {
```

Comment:
Is `Account` the most generic name for this across all cloud provider ?

Reply:
It seems to be account ID for AWS, project ID for GCP and subscription ID for Azure. Account feels pretty generic